modal.js

```diff
@@ -0,0 +1,217 @@
+  const desktopImg = document.createElement('img');
+  desktopImg.src = worksData.desktopImg[i];
+  desktopImg.className = 'project-image hide-pe-mic';
+  mobileImg.alt = worksData.imgAlt[i];
+
+  const descriptionDiv = document.createElement('div');
+  descriptionDiv.className = 'img-description unu';
+
+  const desktopTitle = document.createElement('h3');
+  desktopTitle.className = 'card-title hide-pe-mic-lista';
+  desktopTitle.innerText = worksData.desktopTitle[i];
+
+  const list = document.createElement('ul');
+  list.className = 'card-details';
+
+  const listitem1 = document.createElement('li');
+  listitem1.className = 'hide-pe-mic-lista';
+  listitem1.innerText = worksData.cardDetail1[i];
+
+  const listitem2 = document.createElement('li');
+  listitem2.className = 'hide-pe-mare-lista';
+  listitem2.innerText = worksData.cardDetail2[i];
+
+  const listitem3 = document.createElement('li');
+  listitem3.innerText = '|';
+
+  const listitem4 = document.createElement('li');
+  listitem4.innerText = worksData.cardDetail4[i];
+
+  const listitem5 = document.createElement('li');
+  listitem5.innerText = '|';
+
+  const listitem6 = document.createElement('li');
+  listitem6.innerText = worksData.cardDetail6[i];
+
+  const mobileTitle = document.createElement('h3');
+  mobileTitle.className = 'card-title hide-pe-mare-lista';
+  mobileTitle.innerText = worksData.mobileTitle[i];
+
+  const desktopDescription = document.createElement('p');
+  desktopDescription.className = 'card-description hide-pe-mic-lista';
+  desktopDescription.innerText = worksData.desktopDescription[i];
+
+  const mobileDescription = document.createElement('p');
+  mobileDescription.className = 'card-description hide-pe-mare-lista';
+  mobileDescription.innerText = worksData.mobileDescription[i];
+
+  const techList = document.createElement('ul');
+  techList.className = 'card-list';
+
+  const technologies1 = document.createElement('li');
+  technologies1.className = 'card-tech';
+  technologies1.innerText = worksData.technologies1[i];
+
+  const technologies2 = document.createElement('li');
+  technologies2.className = 'card-tech';
+  technologies2.innerText = worksData.technologies2[i];
+
+  const technologies3 = document.createElement('li');
+  technologies3.className = 'card-tech';
+  technologies3.innerText = worksData.technologies3[i];
+
+  const button = document.createElement('button');
+  button.type = 'button';
+  button.innerText = 'See Project';
+
+  techList.append(technologies1, technologies2, technologies3);
+  list.append(listitem1, listitem2, listitem3, listitem4, listitem5, listitem6);
+  descriptionDiv.append(desktopTitle, list, mobileTitle,
+    desktopDescription, mobileDescription, techList, button);
+  mainDiv.append(mobileImg, desktopImg, descriptionDiv);
+  section.appendChild(mainDiv);
+}
+
+/* Modal window */
+
+const popUp = document.getElementById('popUp');
+const buttons = Array.from(document.querySelectorAll('#works button'));
+let btnIndex;
```

Comment:
Great job with the popup, however, the current design doesn't match the Figma template of your choosing. Kindly fix it. I am sure it would look much better the way it is in Figma design. Also, for the close button of the popup, you can make the cursor as pointer.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/188873802-71837c7c-459b-43fb-b087-b59b64602ba0.png) | ![image](https://user-images.githubusercontent.com/33268927/188873720-2f2990d5-e5b7-4bdf-a73c-afefd4d650aa.png) |

Reply:
Good day! Can you please be a little more specific? Please bear in mind that the modal card is a Square with the size of 1156px by 1163px in Figma. The basic screens today have a 16:9 aspect ratio. So by the maximum width of the design which is 1440px, the maximum height of the monitor would be 810px. This 810px would not accomodate the 1156 pixels required of the modal, so it needed to be adjusted, by reducing the text size. Also there is only one image of this size, so I streched the images to this size so eache project will open a different image.